includes/class-sensei-admin.php

```diff
@@ -1365,16 +1424,21 @@ public function get_lesson_order( $course_id = 0 ) {
 
 	public function save_lesson_order( $order_string = '', $course_id = 0 ) {
 
+		/**
+		 * It is safe to ignore nonce validation here, because this is called
```

Comment:
I wonder if it would make more sense / be possible to do the nonce check inside of this function, so that we can be absolutely certain it was checked and not have to trace back up the code path to verify that. Sorta in the same way that WordPress advocates for late escaping.

Reply:
I see there are many situations like these, so probably not worth worrying about.

Reply:
The problem is that generally the nonce check needs to happen early. In this case, for example, it needs to happen before some of the code in `handle_order_lessons` runs.

In an ideal world, the function directly responsible for handling the request would check the nonce, pull the data out of `$_POST`, and hand the data through to the functions it calls. Having nested functions like this one directly accessing `$_POST` is something I think we should avoid moving forward.

But, for the sake of being pragmatic and not introducing too many backwards-incompatible refactors, I opted to strategically ignore the linter instead, when I felt that it was justified.